src/package.ts

```diff
@@ -304,18 +354,17 @@ export async function updatePackage({
   packageReleasePlan: PackageReleasePlan;
   stderr?: Pick<WriteStream, 'write'>;
 }): Promise<void> {
-  const {
-    package: pkg,
-    newVersion,
-    shouldUpdateChangelog,
-  } = packageReleasePlan;
+  const { package: pkg, newVersion } = packageReleasePlan;
 
   await writeJsonFile(pkg.manifestPath, {
     ...pkg.unvalidatedManifest,
     version: newVersion,
   });
 
-  if (shouldUpdateChangelog) {
-    await updatePackageChangelog({ project, packageReleasePlan, stderr });
-  }
+  await migrateChangelogUnreleasedChangesToRelease({
```

Comment:
Replacing this makes the description for this function a bit outdated. Right now it reads:

```
 * Updates the package as per the instructions in the given release plan by
 * replacing the `version` field in the manifest and adding a new section to the
 * changelog for the new version of the package.
```

It seems like we might want something like:

```
 * Updates the package by replacing the `version` field in the manifest
 * according to the one in the given release plan. Also updates the
 * changelog by migrating changes in the Unreleased section to the section
 * representing the new version.
```